src/components/layout/NavBar.tsx

```diff
@@ -83,6 +84,7 @@ const NavBar = ({ component }: NavBarProps) => {
           </NavDropDown>
           <NavLink href="https://forums.papermc.io/" target="_blank">
             Forums
+            <ExternalUrlIcon className="h-1rem ml-0.5 align-sub" />
```

Comment:
I suggest we replace this with `w-4 h-4 ml-2`

Reply:
I think `ml-2` is too much spacing. `ml-1` looks fine tho